src/parser/classic/modules/items/Potion.ts

```diff
@@ -1,100 +0,0 @@
-import Analyzer, { Options } from 'parser/core/Analyzer';
-import Abilities from 'parser/core/modules/Abilities';
-import Buffs from 'parser/core/modules/Auras';
-import { ThresholdStyle } from 'parser/core/ParseResults';
-import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
-import AbilityTracker from 'parser/shared/modules/AbilityTracker';
-import CastEfficiency from 'parser/shared/modules/CastEfficiency';
-import SpellUsable from 'parser/shared/modules/SpellUsable';
-
-/**
- * Abstract class for potions and healthstone.
- * There are three different categories of pots that share cooldown:
- * Healthstones, health pots and combat pots (DPS, HPS, mana and mitigation).
- * All potions have a 5 minute cooldown.
- *
- * @property {Abilities} abilities
- * @property {Auras} buffs
- * @property {SpellUsable} spellUsable
- * @property {AbilityTracker} abilityTracker
- */
-class Potion extends Analyzer {
```

Comment:
Classic and retail have the same setup for potions now, so there is no reason to maintain this custom class.